### PR TITLE
Add event action seed data and API documentation

### PR DESCRIPTION
**Pull request by navish**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
 
 ## EventAction
 
-> TODO
+`EventAction` groups emergency response actitvities
```
Comment (lykmapipo) [inline docs/api.md]: Please update to be: Define an emergency response activity

File: docs/api.md [hunk 2]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+> Example Request
+
+```curl
+curl --request POST \
+--url https://api.ewea.io/v1/predefines/eventactions \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <apiKey>' \
+--data '{
+"name": { "en": "Working with the Regional Disaster Management Committee to
```
Comment (lykmapipo) [inline docs/api.md]: Please use: Disseminating warning information to the public

File: docs/api.md [hunk 3]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+|  updatedAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was updated. |
+
+> Example Response
+
+```curl
+HTTP/1.1 201 Success
+{
+"_id": "5ddbbc871283e3131b2d41f4",
+"type": "Flood",
+"name": { "en": "Dead Bodies Management" },
```
Comment (lykmapipo) [inline docs/api.md]: Please use: Disseminating warning information to the public

File: docs/api.md [hunk 4]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+"code": "DBM",
+"color": "#F9C5A7"
+"updatedAt": "2019-02-21T13:45:04.340Z",
+"createdAt": "2019-02-21T13:45:04.340Z"
+}
+```
+
+### Retrieve EventAction
+
+To get a event frunction, send a `GET` request to `https://api.ewea.io/v1/predefines/eventactions/:id`.
```
Comment (lykmapipo) [inline docs/api.md]: Please use: To get an event action,

File: docs/api.md [hunk 5]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+|    color     | String |               A color code(in hexadecimal format) used to differentiate event actions visually.                |
+|   function   | String |           Human-translatable-readable name of event function to which the action is categorized into           |
+|  createdAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was created. |
+|  updatedAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was updated. |
+
+> Example Response
+
+```curl
+HTTP/1.1 200 Success
+[{
```
Comment (lykmapipo) [inline docs/api.md]: Please: This is single action response, @BenMaruchu assist

File: docs/api.md [hunk 6]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+    "createdAt": "2019-11-25T11:35:35.518Z"
+  },
+"updatedAt": "2019-12-04T13:42:46.937Z",
+"createdAt": "2019-12-04T13:42:20.186Z"
+}]
+```
+
+### Delete EventAction
+
+To delete existing event group, send a `DELETE` request to `https://api.ewea.io/v1/predefines/eventactions/:id`.
```
Comment (lykmapipo) [inline docs/api.md]: Please change: To delete existing event action,

File: docs/api.md [hunk 7]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+|     code     | String |                             Unique human-readable given code of this event action.                             |
+|    color     | String |               A color code(in hexadecimal format) used to differentiate event actions visually.                |
+|   function   | String |           Human-translatable-readable name of event function to which the action is categorized into           |
+|  createdAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was created. |
+|  updatedAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was updated. |
+
+> Example Response
+
+```curl
+HTTP/1.1 200 Success
```
Comment (lykmapipo) [inline docs/api.md:2714]: Please, this is single response, @BenMaruchu assist

File: docs/api.md [hunk 8]
```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
...
+|     code     | String |                             Unique human-readable given code of this event action.                             |
+|    color     | String |               A color code(in hexadecimal format) used to differentiate event actions visually.                |
+|   function   | String |           Human-translatable-readable name of event function to which the action is categorized into           |
+|  createdAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was created. |
+|  updatedAt   | String | A time value given in ISO8601 combined date and time format that represents when the event action was updated. |
+
+> Example Response:
+
+```curl
+HTTP/1.1 200 Success
```
Comment (lykmapipo) [inline docs/api.md:2765]: Please use simplified format, @BenMaruchu assist

**Review by lykmapipo — CHANGES_REQUESTED** [review]
Please assist in resolve comments

File: docs/api.md (hunk 1 above)
Comment (navish) [reply docs/api.md]: Done

File: docs/api.md (hunk 2 above)
Comment (navish) [reply docs/api.md]: Done

**Review by rijkerd — APPROVED** [review]

**Review by lykmapipo — APPROVED** [review]

**Pull request merged**